CMakeLists.txt

```diff
@@ -145,7 +156,7 @@ add_custom_target(version-info ALL
 	-DCONFIG_FILE=${CMAKE_CURRENT_SOURCE_DIR}/src/core/version.c.in
 	-DOUT_FILE=${CMAKE_CURRENT_BINARY_DIR}/version.c
 	-P ${CMAKE_CURRENT_SOURCE_DIR}/version.cmake
-	WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+	WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
This change is definitely wrong. What was the motivation here?

Reply:
That target will call git from outside of mgba's tree without this change. This is the only thing affected as the other files are referenced by their absolute paths. I have confirmed that this works as intended, unless I am missing something

Reply:
You're right. I was thinking this affected the output directory.